docs/man/docker-run.1.md

```diff
@@ -557,8 +557,8 @@ An MLS example might be:
 
     # docker run --security-opt label:level:TopSecret -i -t rhel7 bash
 
-To disable the security labeling for this container versus running with the
-`--permissive` flag, use the following command:
+To disable the security labeling for this container instead of running with the
```

Comment:
Is this also under SELinux? Not clear. 

Not sure why you are mentioning "instead of running with the `privileged` flag.  Since it takes up so much of the sentence, it might be helpful for the user to understand why this in this context.  If you aren't willing to provide context, I'd omit the clause.